lib/tasks/templates.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Mix.Tasks.Addict.Gen.Templates do
+  use Mix.Task
+  import Mix.Generator
+  alias Addict.Utils
+
+  @shortdoc "Generate templates and view for Addict"
+  @moduledoc """
+  Generates an Addict view and templates
+      mix addict.gen.templates
```

Comment:
Maybe instead of `templates` which colides with the Phoenix templates naming, we could call this `addict.gen.screens`.